common/src/main/java/io/druid/guice/DruidSecondaryModule.java

```diff
@@ -75,24 +76,36 @@ public void configure(Binder binder)
     binder.bind(JsonConfigurator.class).toInstance(jsonConfigurator);
   }
 
-  @Provides @LazySingleton @Json
-  public ObjectMapper getJsonMapper(final Injector injector)
+  @Provides
+  @LazySingleton
```

Comment:
Since we don't need to inject InjectableValues (which is in itself a very confusing thing), we can probably remove this method entirely or at least remove Guice annotations and make it private.